contracts/contracts/poolBooster/PoolBoosterFactorySwapxDouble.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { PoolBoosterSwapxDouble } from "./PoolBoosterSwapxDouble.sol";
+import { AbstractPoolBoosterFactory } from "./AbstractPoolBoosterFactory.sol";
+
+/**
+ * @title Pool booster factory for creating Swapx Ichi pool boosters where both of the
+ *        gauges need incentivizing.
+ * @author Origin Protocol Inc
+ */
+contract PoolBoosterFactorySwapxDouble is AbstractPoolBoosterFactory {
+    uint256 public constant version = 1;
+
+    // @param address _oSonic address of the OSonic token
+    // @param address _governor address governor
+    constructor(address _oSonic, address _governor)
+        AbstractPoolBoosterFactory(_oSonic, _governor)
+    {}
+
+    /**
+     * @dev Create a Pool Booster for SwapX Ichi vault based pool where 2 Bribe contracts need to be
+     *      bribed
+     * @param _bribeAddressOS address of the Bribes.sol(Bribe) contract for the OS token side
+     * @param _bribeAddressOther address of the Bribes.sol(Bribe) contract for the other token in the pool
+     * @param _ammPoolAddress address of the AMM pool where the yield originates from
+     * @param _split 1e18 denominated split between OS and Other bribe. E.g. 0.4e17 means 40% to OS
+     *        bribe contract and 60% to other bribe contract
+     * @param _salt A unique number that affects the address of the pool booster created. Note: this number
+     *        should match the one from `computePoolBoosterAddress` in order for the final deployed address
+     *        and pre-computed address to match
+     */
+    function createPoolBoosterSwapxDouble(
+        address _bribeAddressOS,
+        address _bribeAddressOther,
+        address _ammPoolAddress,
+        uint256 _split,
+        uint256 _salt
+    ) external onlyGovernor {
```

Comment:
Would we be good with having a lower privileges address create pools, to reduce operational burden? Without yield headed in, it should be fine.

Reply:
[For now](https://github.com/OriginProtocol/origin-dollar/pull/2400/files#diff-072de69218436548874e61b3c8f77fd888a65d1f7b66f8ef59cb7534d551acf8R80-R107) the pool boosters are going to be created via the same governance TX that will also enable yield forwarding to the boosters. Which I would argue is the lowest effort operational approach. 
We could make booster Strategizable instead of Governable, adding the extra strategist role. Though I am not sure the added complexity is necessary. Do you think we should also enable Strategists to create pool boosters?

Reply:
It's fine as you have it with the create2.